### PR TITLE
Link to systemctl update docs in rpm/deb install instructions

### PR DESCRIPTION
**Pull request by feorlen**
The update instructions in the "You are running an older version" banner are not correct for systemd managed deployments (also the process recommended for rpm/deb installs). 

Link to the correct update procedure in the install docs. This is not intended to be comprehensive, just pointers for future-admin who might have forgotten.

Staged:
http://192.241.195.202:9000/staging/deb-upgrades/linux/html/operations/install-deploy-manage/deploy-minio-single-node-single-drive.html#download-the-minio-server

### CODE REVIEW

File: source/includes/common-minio-mc.rst [hunk 1]
```diff
@@ -42,9 +42,10 @@ commands *may* work as documented, any such usage is at your own risk.
 
 - Brackets ``[]`` indicate optional parameters. 
 - Parameters sharing a line are mutually dependent.
-- Parameters seperated using the pipe ``|`` operator are mutually exclusive.
+- Parameters separated using the pipe ``|`` operator are mutually exclusive.
```
Comment (feorlen) [inline source/includes/common-minio-mc.rst:45]: Drive-by spelling fix.

File: source/includes/common-minio-mc.rst [hunk 2]
```diff
@@ -42,9 +42,10 @@ commands *may* work as documented, any such usage is at your own risk.
...
 
 Copy the example to a text editor and modify as-needed before running the
 command in the terminal/shell.
+You may need to use ``sudo`` if your user does not have write permissions for the path where ``mc`` is installed.
```
Comment (feorlen) [inline source/includes/common-minio-mc.rst:49]: Another helpful reminder, since we recommend updating `mc` after updating MinIO.

Is this a good place to add this text?

File: source/includes/linux/common-installation.rst [hunk 3]
```diff
@@ -4,8 +4,8 @@ The following tabs provide examples of installing MinIO onto 64-bit Linux
 operating systems using RPM, DEB, or binary. The RPM and DEB packages
 automatically install MinIO to the necessary system paths and create a
 ``systemd`` service file for running MinIO automatically. MinIO strongly
-recommends using RPM or DEB installation routes.
-
+recommends using RPM or DEB installation routes. To update deployments
```
Comment (feorlen) [inline source/includes/linux/common-installation.rst:7]: One of two proposed locations for the update link.

The title of the linked section references `systemctl` so I used that. Using both `systemd` and `systemctl` here seems a bit confusing, but so would calling it `systemd` and landing on a page about `systemctl`. Suggestions welcome.

Comment (ravindk89) [reply source/includes/linux/common-installation.rst:7]: I think just calling it `systemctl` is fine since that is the proc that actually handles the automation.

Maybe for the graph on 4-6 we could state something like "create a ``minio`` service for ``systemctl``"

Comment (feorlen) [reply source/includes/linux/common-installation.rst]: Went with both your suggestions. And joined sentences since I was in there messing with existing text anyway.

What about the other instances of this same original paragraph? Edit or leave them alone?

Comment (ravindk89) [reply source/includes/linux/common-installation.rst]: :thinking: if you want to expand scope and hit the other ones, go for it. Might as well while we're here, so to speak.

File: source/includes/linux/deploy-standalone.rst [hunk 4]
```diff
@@ -1,11 +1,10 @@
 1) Download the MinIO Server
 ----------------------------
 
-The following tabs provide examples of installing MinIO onto 64-bit Linux
-operating systems using RPM, DEB, or binary. The RPM and DEB packages
-automatically install MinIO to the necessary system paths and create a
-``systemd`` service file for running MinIO automatically. MinIO strongly
-recommends using RPM or DEB installation routes.
+The following tabs provide examples of installing MinIO onto 64-bit Linux operating systems using RPM, DEB, or binary.
+The RPM and DEB packages automatically install MinIO to the necessary system paths and create a ``minio`` service for ``systemctl``.
```
Comment (ravindk89) [inline source/includes/linux/deploy-standalone.rst:5]: completely optional but it occurs to me we could just move this graf into the tab set and specify RPM or DEB respectively.

Then it only shows up when selecting the relevant tab. For binary builds, the user doesn't care about this information.

**Review by ravindk89 — APPROVED** [review]
LGTM mod one suggestion

File: source/includes/linux/deploy-standalone.rst (hunk 4 above)
Comment (feorlen) [reply source/includes/linux/deploy-standalone.rst:5]: After discussion, decided to leave it where it is so it's more visible. While the audience I had in mind wanted RPM/DEB and only got `systemctl` because we recommended it, there are other possible ways to trip over this.
